graphistry/kepler.py

```diff
@@ -0,0 +1,841 @@
+"""
```

Comment:
can we move into something like https://github.com/graphistry/pygraphistry/tree/master/graphistry/plugins ?